portal/api-tests/static-routes.api-test.js

```diff
@@ -39,10 +39,12 @@ describe('/.well-known/security.txt', () => {
       // Act
       const response = await get('/.well-known/security.txt');
       // Only get the YYYY-MM-DD
-      const todayDate = getISO8601().split('T')[0];
+      const addOneYear = addYear(1);
+      const expiry = getISO8601(addOneYear);
+      const oneYearFromToday = expiry.split('T')[0];
```

Comment:
I'm slightly confused as in the `static.js` file we're displaying the date as an ISO string, whereas in this test file we seem to just be displaying YYYY-MM-DD - is there a reason for the difference?

Either way, this could also be written as:

```
const oneYearInFutureDate = addYear(1);
const oneYearInFutureFormatted = format(oneYearInFutureDate, "yyyy-MM-dd")
```